tests/framework-tests/preferences/data/textSize_fr.json

```diff
@@ -0,0 +1,6 @@
+{
+    "textSizeLabel": "Text Size",
```

Comment:
We should put the contents of these _fr message files into at least some kind of French, so that we don't become confused.